search/src/main/scala/uk/ac/wellcome/platform/api/search/Router.scala

```diff
@@ -45,63 +45,70 @@ class Router(
-              },
-              path("works" / Segment) { id: String =>
-                Try { CanonicalId(id) } match {
-                  case Success(workId) =>
-                    SingleWorkParams.parse {
-                      worksController.singleWork(workId, _)
-                    }
-
-                  case _ => notFound(s"Work not found for identifier $id")
-                }
-              },
-              path("images") {
-                MultipleImagesParams.parse { imagesController.multipleImages }
-              },
-              path("images" / Segment) { id: String =>
-                Try { CanonicalId(id) } match {
-                  case Success(imageId) =>
-                    SingleImageParams.parse {
-                      imagesController.singleImage(imageId, _)
-                    }
-
-                  case _ => notFound(s"Image not found for identifier $id")
-                }
-              },
-              path("context.json") {
-                getFromResource("context-v2.json")
-              },
-              path("swagger.json") {
-                swagger
-              },
-              path("search-templates.json") {
-                getSearchTemplates
-              }
-            )
-          },
-          pathPrefix("management") {
-            concat(
-              path("healthcheck") {
-                get {
-                  complete("message" -> "ok")
-                }
-              },
-              path("clusterhealth") {
-                getClusterHealth
-              }
-            )
+  def searchRoutes: Route = concat(
+    path("works") {
+      MultipleWorksParams.parse { worksController.multipleWorks }
+    },
+    path("works" / Segment) { id: String =>
+      Try { CanonicalId(id) } match {
+        case Success(workId) =>
+          SingleWorkParams.parse {
+            worksController.singleWork(workId, _)
           }
-        )
+
+        case _ => notFound(s"Work not found for identifier $id")
       }
+    },
+    path("images") {
+      MultipleImagesParams.parse { imagesController.multipleImages }
+    },
+    path("images" / Segment) { id: String =>
+      Try { CanonicalId(id) } match {
+        case Success(imageId) =>
+          SingleImageParams.parse {
+            imagesController.singleImage(imageId, _)
+          }
+
+        case _ => notFound(s"Image not found for identifier $id")
+      }
+    },
+    path("context.json") {
+      getFromResource("context-v2.json")
+    },
+    path("swagger.json") {
+      swagger
+    },
+    path("search-templates.json") {
+      getSearchTemplates
+    }
+  )
+
+  def legacyRoutes: Route = pathPrefix("catalogue") {
+    pathPrefix("v2") {
+      concat(
+        searchRoutes,
+        pathPrefix("management") {
+          concat(
+            path("healthcheck") {
+              get {
+                complete("message" -> "ok")
+              }
+            },
```

Comment:
Do we need a healthcheck path in the non-legacy routes?

Reply:
No - the healthcheck is for the service itself so stays at the root. Also, more pragmatically, I don't believe we ever use it 🙃